homeassistant/components/voice_assistant/websocket_api.py

```diff
@@ -19,9 +30,11 @@ def async_register_websocket_api(hass: HomeAssistant) -> None:
 @websocket_api.websocket_command(
     {
         vol.Required("type"): "voice_assistant/run",
+        vol.Required("start_stage"): str,
+        vol.Required("end_stage"): str,
```

Comment:
We should limit this to the enum. I think that you can just put the enum as validation ?